graphene/types/schema.py

```diff
@@ -164,8 +174,7 @@ def create_scalar(graphene_type):
             parse_literal=getattr(graphene_type, "parse_literal", None),
         )
 
-    @staticmethod
```

Comment:
not sure it is needed for it to be a static method... but I guess it also does not need `self` so it could totally be static. And there is no real downside of having it as a static method. (also that needs to be consistent with `create_scalar`.
I would revert this change in my opinion...

Reply:
I do need `self` to add the prefix actually: https://github.com/graphql-python/graphene/pull/1466/files#diff-f066ce1b84329fbfb7cd0e5d5027a51f68e14677754aab172fbc91dea42c519dR205

Reply:
(I've removed the `@static_method` for `create_scalar` too)

Reply:
makes sense 👍

Reply:
This makes it a breaking change with potential implications for downstream libs. Is there any way to avoid that?

Reply:
I'm thinking something like an optional arg would easily prevent that. We could add a note to overhaul the syntax in the next breaking release. But I'm personally against pushing even small breaking changes in patches/minors. Open to discuss this though!

Reply:
That's totally fair, I've updated the code let me know

Reply:
Looks good now, thanks!